tests/scripts/thread-cert/Cert_8_1_01_Commissioning.py

```diff
@@ -65,7 +65,7 @@ def test(self):
 
         self.nodes[JOINER].interface_up()
         self.nodes[JOINER].joiner_start('openthread')
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
Looking at the logs, I was able to track down bit gaps in time after the following messages:

```
[otCore]    MBED found ecjpake kkpp extension
```

I looked at what it does, and it allocates and frees a TON of things. I happen to have (for debug mode) code in each alloc/free to do additional tracking to make sure there weren't any leaks, and I guess that was enough a of a perf hit to cause these delays. Because once I removed them, the timing looks like this:

```
10:11:30.862    16A8    [otNodeApi]    ---> otNodeJoinerStart [2] openthread <NULL>
10:11:30.862    0AB4       [otCore]    ---> Thread::MeshCoP::Joiner::Start
10:11:30.862    0AB4       [otCore]    MLE Sent discovery request
10:11:30.863    0DF0       [otCore]    MLE Received discovery request
10:11:30.863    0DF0       [otCore]    MLE Sent discovery response
10:11:30.863    0AB4       [otCore]    MLE Handle discovery response
10:11:34.435    0AB4       [otCore]    COAP DTLS started
10:11:36.388    0AB4       [otCore]    COAP Generated KEK
10:11:36.733    0AB4       [otCore]    COAP Sent joiner finalize
10:11:36.734    0DF0       [otCore]    COAP receive joiner finalize 1
10:11:36.735    0DF0       [otCore]    COAP Received kek
10:11:36.737    0AB4       [otCore]    COAP join success!
```